fmriprep/interfaces/bids.py

```diff
@@ -170,6 +170,9 @@ class DerivativesDataSinkInputSpec(BaseInterfaceInputSpec):
 
 class DerivativesDataSinkOutputSpec(TraitedSpec):
     out_file = OutputMultiPath(File(exists=True, desc='written file path'))
+    compression = traits.List(
+        traits.Bool, desc='whether ``in_file`` was compressed/uncompressed '
+                          'or `it was copied directly.')
```

Comment:
I would just use `OutputMultiPath`, here. There's no good reason to make it a list when `out_file` isn't.